crates/types/src/rpc/transactions/broadcasted_invoke_transaction.rs

```diff
@@ -12,6 +19,15 @@ use crate::felt::{
 use crate::rpc::transactions::invoke_transaction_v1::InvokeTransactionV1;
 use crate::rpc::transactions::BroadcastedTransactionCommon;
 
+const QUERY_VERSION_OFFSET: FieldElement = FieldElement::from_mont([
+    18446744073700081665,
+    17407,
+    18446744073709551584,
+    576460752142434320,
+]);
+
+const SUPPORTED_TX_VERSION: FieldElement = FieldElement::ONE;
```

Comment:
Sorry for coming back to this, but instead of defining a new constant, couldn't we import `SUPPORTED_TX_VERSION` from the file where it's already defined and here use `FieldElement::from(SUPPORTED_TX_VERSION)`.

Reply:
no, because the already defined constant is in `starknet` crate and the new one is in `types` crate. The chain of imports is types -> starknet